### PR TITLE
feat(Card Thumbnail): implements native lazy load

### PR DESCRIPTION
**Pull request by rapahaeru**
## Description
task: https://jirasoftware.catho.com.br/browse/QTM-178
Issue: https://github.com/catho/quantum/issues/130

⚠️ this new attribute only will show in the  when the styled-components is updated.
In actual version `4.1.3`, this attribute did not exist and the styled removes the attributes that not recognise.
But, with this PR, when the styled-components is update, the attribute will show and works.

The problem in styled was solved here https://github.com/styled-components/styled-components/issues/2552

## Setup
to view the components behavior, run `yarn storybook`

### CODE REVIEW

File: components/Card/sub-components/Thumbnail.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import styled from 'styled-components';
 
 import { colors } from '../../shared/theme';
 
-const Image = styled.img`
+const Image = styled.img.attrs({ loading: 'lazy' })`
```
Comment (renatogalvones) [inline components/Card/sub-components/Thumbnail.jsx]: Doubt: Could this be solved by writing it inline like
```
const Thumbnail = ({ ...props }) => <Image {...props} loading="lazy" />;
```
?

Comment (rapahaeru) [reply components/Card/sub-components/Thumbnail.jsx]: the effect is the same.
I tested it both ways...

Comment (renatogalvones) [reply components/Card/sub-components/Thumbnail.jsx]: With this inline solution we still have to upgrade the styled components lib?

Comment (rapahaeru) [reply components/Card/sub-components/Thumbnail.jsx]: unfortunately, yes!

**Pull request merged**